common/pom.xml

```diff
@@ -96,6 +96,18 @@
             <groupId>org.jboss.spec.javax.annotation</groupId>
             <artifactId>jboss-annotations-api_1.2_spec</artifactId>
         </dependency>
+
+        <dependency>
+          <groupId>javax.json</groupId>
+          <artifactId>javax.json-api</artifactId>
+          <version>1.0</version>
```

Comment:
I believe (and hope) this is a test dependency. It should be scoped as such. Also it's better (and easier for prod) is you put the actual version in the top level pom <dependencyManagement> section.